applications/asset_tracker_v2/src/modules/debug_module.c

```diff
@@ -195,10 +195,24 @@ static void watchdog_handler(const struct watchdog_evt *evt)
 }
 #endif /* defined(CONFIG_WATCHDOG_APPLICATION) */
 
-static void send_memfault_data(void)
+/**
```

Comment:
No need to change here, but this could be a single `*`, as it should not be parsed by Doxygen (probably won't be anyway, as this is a .c fil)